gtests/net/mptcp/dss/mpc_with_data_server_window_close.pkt

```diff
@@ -0,0 +1,31 @@
+// connection initiated by packetdrill
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0     setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
+
++0     `nstat -n`
```

Comment:
(details: it looks like you don't use `nstat` below, maybe you don't need this line?)

Reply:
(or maybe the idea is to check that the new counter has been incremented at the end, similar to what we do in other tests like `dss/mpc_with_data_server.pkt`?)